docs/src/main/sphinx/connector/postgresql.md

```diff
@@ -51,6 +51,18 @@ determine the user credentials for the connection, often a service user. You can
 use {doc}`secrets </security/secrets>` to avoid actual values in the catalog
 properties files.
 
+### Access to system tables
+
+The PostgreSQL connector supports reading PostgreSQL system tables, such as `pg_catalog`.
+The functionality is disabled by default, and can be enabled using the ``postgresql.include-system-tables`` configuration property.
+
+```
+SHOW SCHEMAS FROM pg_catalog;
```

Comment:
sorry .. I am confused .. in what schemas are the system tables exposed when this is enabled? 

Also .. should the example not be

show tables from example.notsurewhatschema 

or 

describe example.notsurewhatschema.pg_catalog ?